packages/message-input/src/vaadin-message-input-mixin.js

```diff
@@ -111,8 +116,17 @@ export const MessageInputMixin = (superClass) =>
             }
           });
 
-          const input = textarea.inputElement;
-          input.removeAttribute('aria-labelledby');
+          let input = textarea.inputElement;
+
+          // With Lit version, input element renders asynchronously and it will
+          // override the `rows` attribute set to `1` in the `minRows` observer.
+          // Workaround by creating a custom textarea instead to not update it.
+          // TODO: needs https://github.com/vaadin/web-components/pull/8168
+          if (!input) {
+            input = document.createElement('textarea');
+            input.setAttribute('slot', 'textarea');
+            textarea.appendChild(input);
+          }
```

Comment:
Alternative workaround:
```js

if (textarea.performUpdate) {
  textarea.performUpdate();
  textarea.performUpdate();
}

const input = textarea.inputElement;
```

Reply:
Thanks, this makes more sense as we might be able to eventually remove second update once #8168 is merged. Updated to use this workaround and changed the comment accordingly.